src/content/en/tools/chrome-devtools/index.markdown

```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
 order: 3
 ---
 
-<div class="wf-subheading wf-devtools-header">
-  <div class="page-content">
-      <h1><img src="images/chrome-devtools-logo.png" alt="">Chrome DevTools</h1>
-  </div>
-  <div class="page-content mdl-grid">
-    <div class="mdl-cell mdl-cell--6-col">
-      <p>The Chrome DevTools are a set of web authoring and debugging tools built into Google Chrome. Use the DevTools to efficiently track down layout issues, set JavaScript breakpoints, and get insights for code optimization.</p>
-      <div class="note">
-        Note: Use <a href="https://tools.google.com/dlpage/chromesxs">Chrome Canary</a> for the latest DevTools.
-      </div>
+<div class="wf-devtools-wrapper">
+
+  <div class="wf-subheading wf-devtools-header">
+    <div class="page-content">
+        <h1><img src="images/chrome-devtools-logo.png" alt="">Chrome DevTools</h1>
     </div>
-    <div class="mdl-cell mdl-cell--6-col">
-      <p class="access">To access the DevTools:</p>
-      <ul>
-        <li>Select <strong>Tools > Developer Tools</strong> from the Chrome Menu.</li>
-        <li>Right-click on any page element and select Inspect Element.</li>
-        <li>Use <kbd class="kbd">Ctrl</kbd>+<kbd class="kbd">Shift</kbd>+<kbd class="kbd">I</kbd> (or <kbd class="kbd">Cmd</kbd>+<kbd class="kbd">Opt</kbd>+<kbd class="kbd">I</kbd> on Mac) (<a href="https://developer.chrome.com/devtools/docs/shortcuts">more shortcuts</a>)</li>
-      </ul>
+    <div class="page-content mdl-grid">
+      <div class="mdl-cell mdl-cell--6-col">
+        <p>The Chrome DevTools are a set of web authoring and debugging tools built into Google Chrome. Use the DevTools to iterate, debug and profile your site.</p>
+        <div class="note">
+          <a href="https://tools.google.com/dlpage/chromesxs">Chrome Canary</a> always has the latest DevTools.
+        </div>
+      </div>
+      <div class="mdl-cell mdl-cell--6-col">
+        <ul>
+          <li>Select <strong>More Tools > Developer Tools</strong> from the Chrome Menu.</li>
+          <li>Right-click on a page element and select Inspect</li>
+          <li>Use <kbd class="kbd">Ctrl/Cmd</kbd>+<kbd class="kbd">Shift</kbd>+<kbd class="kbd">I</kbd> (<a href="https://developer.chrome.com/devtools/docs/shortcuts">more shortcuts</a>)</li>
```

Comment:
Yes. It was historically so through all of our docs for some reason and got ported over this way. We can probably fix this in the future but this PR is not the appropriate place.